x-pack/plugins/fleet/public/plugin.ts

```diff
@@ -141,11 +144,16 @@ export class FleetPlugin implements Plugin<FleetSetup, FleetStart, FleetSetupDep
       euiIconType: 'logoElastic',
       mount: async (params: AppMountParameters) => {
         const [coreStartServices, startDepsServices, fleetStart] = await core.getStartServices();
+        const cloud =
+          deps.cloud && startDepsServices.cloud
+            ? { ...deps.cloud, ...startDepsServices.cloud }
+            : undefined;
```

Comment:
(outside of the scope of this PR) logic in `mount` functions should not need to access the setup contract of their dependencies. We'd probably want to expose the cloud config from its start contract too to avoid that kind of things.